src/aleph/vm/hypervisors/firecracker/microvm.py

```diff
@@ -324,7 +324,7 @@ def enable_file_rootfs(self, path_on_host: Path) -> Path:
             rootfs_filename = Path(path_on_host).name
             jailer_path_on_host = f"/opt/{rootfs_filename}"
             try:
-                os.link(path_on_host, f"{self.jailer_path}/{jailer_path_on_host}")
+                os.symlink(path_on_host, f"{self.jailer_path}/{jailer_path_on_host}")
```

Comment:
I think that this needs to be a hard link, because for Firecracker instances, the rootfs file is a devmapper device, and using jailer the unique way to make it work is doing a hard link, not a symlink.